pages/property-filter/async-loading.integ.page.tsx

```diff
@@ -25,7 +26,7 @@ const filteringProperties = [
     groupValuesLabel: `Label values`,
   },
 ] as const;
-const serverFilteringOptions = range(1000).map(value => ({
+const filteringOptions = range(1000).map(value => ({
```

Comment:
In the original page implementation all 1000 options are resolved as a single page. I updated this to be 10 pages with 100 items each instead to showcase progressive loading. That revealed what I would consider a bug: once a property is selected the fetch is called twice (for the first and the second page already). I updated the integration test accordingly and added a comment.